docs/examples/all-ethernet-up/current.yaml

```diff
@@ -0,0 +1,15 @@
+interfaces:
+- accept-all-mac-addresses: false
+  lldp:
+    enabled: false
+  mac-address: 52:55:00:D1:55:02
+  name: eth0
+  state: up
+  type: ethernet
+- accept-all-mac-addresses: false
+  lldp:
+    enabled: false
+  mac-address: 52:55:00:D1:56:02
+  name: eth1
+  state: down
+  type: ethernet
```

Comment:
Would it make sense to add for example:

```yaml
- name: veth1
  type: veth
```
to exercise the fix in this PR - having an interface that doesn't have the `lldp` map specified.